.github/README.md

```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following *steps in this order*:
```

Comment:
Well done on creating a readme file for the project. Kindly utilize the new [readme template](https://github.com/microverseinc/readme-template) by Microverse for documentation. Kindly note that in future projects, it'll be required to utilize this template for your readme in order to receive a review.

Remember to customize the template to your project.